python/lsst/ts/criopy/gui/actuatorsdisplay/fcu_item.py

```diff
@@ -0,0 +1,149 @@
+    showing it is the selected actuator.
+
+    Parameters
+    ----------
+    fcu : `FCUData`
+        Row from force actuator table.
+    data : `float`
+        Data associated with the actuator (actual force, calculated force, ..).
+    data_index : `int`
+        Index in value arrays. Points to selected actuator value.
+    scale: `object`
+        Object providing get_brush(value) method.
+    state : `int`
+        Force Actuator state. 0 for inactive/unused, 1 for active OK, 2 for
+        active warning.
+    kind : `FASelection`
+        FA kind - normal, selected or neighbour of selected.
+    """
+
+    def __init__(self, fcu: FCUData, state: DataItemState):
+        super().__init__(state)
+        self.fcu = fcu
+        # actuator position
+        self._center = QPointF(fcu.x_position * 1000.0, -fcu.y_position * 1000.0)
+        # scale. Provides get_brush(data) object, returning brush to fill data
+        # scalign factor. The actuator default size is 20x20 units. As
+        # actuators are placed on mirror, the size needs to be adjusted to show
+        # properly actuator on display in e.g. mm (where X and Y ranges are
+        # ~-4400 .. +4400).
+        self._scale_factor = 25
+
+    def set_color_scale(self, scale: GaugeScale) -> None:
+        """Set actuator data display scale. This is used for setting display
+        color and formatting values.
+
+        Parameters
+        ----------
+        scale : `object`
+            Scaling object. Should provide formatValue() and get_brush()
+            methods."""
+        self._color_scale = scale
+        self.update()
+
+    def boundingRect(self) -> QRect:
+        """Returns rectangle occupied by drawing. Overridden method."""
+        return QRect(
+            self._center.x() - 10 * self._scale_factor,
+            self._center.y() - 10 * self._scale_factor,
+            20 * self._scale_factor,
+            20 * self._scale_factor,
+        )
+
+    def paint(
+        self, painter: QPainter, option: QStyleOptionGraphicsItem, widget: QWidget
+    ) -> None:
+        """Paint actuator. Overridden method."""
+        assert self._color_scale is not None
+
+        palette = QGuiApplication.palette()
+        if not self.isEnabled():
+            palette.setCurrentColorGroup(palette.Inactive)
+
+        painter.setRenderHint(QPainter.Antialiasing)
+        painter.setRenderHint(QPainter.SmoothPixmapTransform)
+
+        lineStyle = Qt.SolidLine if self.isEnabled() else Qt.DotLine
+        painter.setPen(QPen(Qt.red, self._scale_factor, lineStyle))
+
+        brush = self._color_scale.get_brush(self._data)
+        painter.setBrush(brush)
+
+        # draw actuator, write value
+        painter.drawRect(
+            self._center.x() - 8 * self._scale_factor,
+            self._center.y() - 8 * self._scale_factor,
+            16 * self._scale_factor,
+            16 * self._scale_factor,
+        )
```

Comment:
Please help to explain the magic numbers of 8 and 16 in comment. The same comments for other magic numbers.